peregrine/iqgen/bits/doppler_const.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (C) 2016 Swift Navigation Inc.
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+"""
+The :mod:`peregrine.iqgen.doppler_const` module contains classes and functions
+related to generation of zero-doppler signals.
+
+"""
+
+import scipy.constants
+from peregrine.iqgen import if_iface
+
+class Doppler(object):
+
+  NAME = "ConstDoppler"
+
+  '''
+  Doppler control for a signal source that moves with a constant speed.
+  '''
+  def __init__(self, distance0_m, speed_mps):
+    '''
+    Constructs doppler control object for linear movement.
+
+    Parameters
+    ----------
+    distance0_m : float
+      Initial distance to satellite at time 0 in meters for computing signal
+      phase and delay.
+    speed_mps : float
+      Speed of satellite in meters per second.
```

Comment:
Could you clarify this comment a bit - is this speed along the vector to the satellite?